lib/std/zig/system.zig

```diff
@@ -612,6 +612,7 @@ pub const NativeTargetInfo = struct {
             error.AccessDenied => return error.GnuLibCVersionUnavailable,
             error.FileSystem => return error.FileSystem,
             error.SymLinkLoop => return error.SymLinkLoop,
+            error.NotLink => return error.NotLink,
```

Comment:
I think the quickest fix would be to just do:

```suggestion
            error.NotLink => return error.GnuLibCVersionUnavailable,
```

Reply:
thx, but I can not build the zig and I have to see the result in CI which leads to a waste of resources

Reply:
@squeek502 thank you. build is successful!